src/utils/translate/en.json

```diff
@@ -2,22 +2,18 @@
   "95%ConfidenceInterval": "95% Confidence Interval",

```

Comment:
What's the purpose of leaving this old en.json file?

Reply:
Because you need it to hold the english strings

Reply:
Uhh but we also have another file at path `src/utils/translate/newjson/en.json`, what's the distinction here

Reply:
that was not supposed to be there, I have deleted it